### ISSUE TITLE
Fetch interface list from `holod`

### ISSUE THREAD

**Issue by frederic-loui** [holo-routing/holo#21]
For now when we configure `holo` via `holo-cli`, `interface` is entered in a free form.

It would be ideal to be able to configure an interface from `holo-cli` :

1. From a list of available physical interfaces 
    
2.  From a list of interface category of interface: `te-tunnel`, `tunnel`, `bundle`, `hairpin`, etc 
3.  Restrict sub-interface creation with `.` or `unit` notation form to:
     3.1. Listed physical interface in `1`  
     3.2. Or logical interface listed in `2`

This will enforce configuration consistency.

For `1`, it means that `holo-cli` should be able to query `holod`->`interface list`

**Comment by rwestphal** [comment]
> For `1`, it means that `holo-cli` should be able to query `holod`->`interface list`

Correct. I think the best way to do that would be by adding a new gRPC RPC like `SystemInfo`. That RPC would return the list of available interfaces at the OS-level, and maybe additional data as well, such as the available tunneling technologies.

I'll transfer this issue to the *holo-cli* repo since it's primarily CLI work.
